docs/source/manual/file_formats/pcf_file.rst

```diff
@@ -5,11 +5,13 @@ Pin Constraints File (.pcf)
 
 .. note:: This file is in a different usage than the Pin Constraints File in XML format (see details in :ref:`file_format_pin_constraints_file`)
 
-The PCF file is the file which **users** should craft to assign their I/O constraints
+The PCF file is the file which **users** should craft to assign their I/O constraints.
 
-An example of the file is shown as follows.
+Standard pcf command examples
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+An example of a standard PCF file is shown as follows:
 
-.. code-block:: xml
+.. code-block:: pcf
```

Comment:
I think the ``pcf`` format is not supported by RST code-block. Please double-check